### PR TITLE
Enrollment broken due to bad merge conflictr resolution

### PR DESCRIPTION
**Pull request by btry**
A cherry pick generated a conflict. I introduced an error while solving it. This PR should fix the bug.

### Checklist

Please check if your PR fulfills the following specifications:

- [ ] Tests for the changes have been added
- [ ] Docs have been added/updated

### Estimated time

|Assignee|:tomato:|
|:---|:---:|
|@btry |1|

### References

Closes #N/A
Related #N/A
Depends on #N/A

### CODE REVIEW

File: inc/agent.class.php [hunk 1]
```diff
@@ -1172,13 +1172,6 @@ protected function enrollByInvitationToken($input) {
          return false;
       }
 
-      if (empty($serial) && empty($uuid)) {
```
Comment (DIOHz0r) [inline inc/agent.class.php]: Why did you remove this check if they are mandatory?

Comment (btry) [reply inc/agent.class.php]: i tell you in live
